pkg/api/response/response.go

```diff
@@ -253,8 +253,9 @@ func Error(status int, message string, err error) *NormalResponse {
 
 // Err creates an error response based on an errutil.Error error.
 func Err(err error) *NormalResponse {
-	grafanaErr := &errutil.Error{}
-	if !errors.As(err, grafanaErr) {
+	grafanaErr, err := errutil.ErrorFrom(err)
```

Comment:
NB: We do this operation for other styles of HTTP handlers, we should use `ErrorFrom` in more places.